docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -205,8 +208,8 @@ currently include:
    this node, kubelet removes this taint.
 
 When the `TaintBasedEvictions` alpha feature is enabled (you can do this by
-including `TaintBasedEvictions=true` in `--feature-gates`, such as
-`--feature-gates=FooBar=true,TaintBasedEvictions=true`), the taints are automatically
+including `TaintBasedEvictions=true` in `--feature-gates` for Kubernetes controller manager,
```

Comment:
isn't feature-gates a global setting that should be synced across all components?

Reply:
No, it's per components.

Reply:
@tengqm :wave: Are we OK to merge?

Reply:
This PR does no harm. I agree it can be merged.
What I wanted to point out is that --feature-gates is a cross-component flag. Sometimes it has to be  specified on more than one component, sometimes it only requires to be set on a single one. There seems to be no guidance on which feature needs to be set on which component(s). So, in practice, I'd suggest users to update feature-gates on all components to the same settings.
With that said, calling out a specific component is not harmful.